style.css

```diff
@@ -5,7 +5,6 @@
   margin: 0;
   padding: 0;
   list-style: none;
-  text-decoration: none;
 }
 
 body {
```

Comment:
Your project image seemed to be stretched out and is wider than the design width of the Figma template. ↔️
I suggest that you adjust the width and height as seen below and make some margin changes around it to fit the page ♻️